src/main/resources/io/jenkins/plugins/designlibrary/Buttons/index.jelly

```diff
@@ -97,5 +97,25 @@ THE SOFTWARE.
       <li>text - the text to be copied into the clipboard.</li>
       <li>message - message to be displayed upon the successful copying into the clipboard </li>
     </ul>
+
+    <h2>Help button</h2>
+    <div class="app-component-sample">
+      <t:help href="https://www.jenkins.io" tooltip="${%Additional information on buttons}" />
+    </div>
+    <pre>
+      <code class="language-xml">
+        &lt;t:help href="https://www.jenkins.io" tooltip="${%Additional information on buttons}" /&gt;
```

Comment:
unfortunately this won't render properly.

prism seems to expand any self closing tags, dunno why.

It will end up like:

`<t:help href="https://www.jenkins.io" tooltip="${%Additional information on buttons}"></t:help>`

I've been moving these to remote samples to work around